### DISCUSSION TITLE
Should the polybar bar config docs document px and pt values

### DISCUSSION THREAD

**Discussion by dceluis** [polybar/polybar#2834] [Ideas]
Hi,

On https://github.com/polybar/polybar/wiki/Configuration#bar-settings
the docs show: 
```
; Width and height of the bar window.
; Supports any percentage with offset.
; For 'width', the percentage is relative to the monitor width and for 'height'
; relative to the monitor height
width =
height =
```
https://github.com/polybar/polybar/blob/master/doc/config.ini#L30 though makes use of pt values, in addition to the documented percentage values allowed.
I tried px values, and they are also allowed, so I volunteer to add a note to these to the polybar wiki on bar configuration.
So that it's clear that these are available.

**Comment by patrick96** [comment]
Technically this is already documented in the wiki: https://github.com/polybar/polybar/wiki/Formatting#sizes--spacing

"Percentage with offset" means either a single percentage value, a percentage value with an offset, or just an offset. Just an offset would correspond to simply using px and pt values.

The wording probably isn't great, I wanted to avoid having to explain exactly which values are acceptable by simply saying "percentage with offset". Maybe we can say something like "percentage with offset or an extent value".

Once we move all the configuration stuff into [readthedocs](https://polybar.readthedocs.io/en/stable/), we will also be able to crosslink so it becomes evident that they can look up what "percentage with offset" and "extent value" exactly means.

The wiki is editable by everyone, so if you think you can improve it, feel free to edit. Thanks!

**Reply by dceluis** [reply]
Got it, thank you. Ill make the small change "add extant" and wait for the migration to readthedocs that will make the whole thing even better
